internal/ad/registry/registry.go

```diff
@@ -196,10 +196,11 @@ func readPolicy(r io.Reader) (entries []policyRawEntry, err error) {
 		return nil, fmt.Errorf("file header: %x%x", header.Signature, header.Version)
 	}
 
-	sectionStart := []byte{'[', 0}     // [ in UTF-16 (little endian)
-	sectionEnd := []byte{0, 0, ']', 0} // \0] in UTF-16 (little endian)
+	sectionStart := []byte{'[', 0}                // [ in UTF-16 (little endian)
+	sectionEndNullChar := []byte{0, 0, ']', 0}    // \0] in UTF-16 (little endian)
+	sectionEndSemicolon := []byte{';', 0, ']', 0} // ;] in UTF-16 (little endian)
```

Comment:
nitpick: I would not name it this way. It’s not really the semi colon which is part of the end section, it’s the delimiter of the previous section. So maybe the name would be more in line with sectionEndNoNullChar, and then, the comment explains why we take the semi-colon as part of the sequence.